runtime/Go/antlr/lexer_atn_simulator.go

```diff
@@ -311,7 +311,7 @@ func (l *LexerATNSimulator) accept(input CharStream, lexerActionExecutor *LexerA
 }
 
 func (l *LexerATNSimulator) getReachableTarget(trans Transition, t int) ATNState {
-	if trans.Matches(t, 0, LexerMaxCharValue) {
+	if trans.Matches(t, 0, 0xFFFE) {
```

Comment:
@millergarym you need to pull from master as it looks like you are undoing our unicode 32 stuff.